packages/react-devtools-shared/src/__tests__/inspectedElementContext-test.js

```diff
@@ -64,6 +66,7 @@ describe('InspectedElementContext', () => {
       .StoreContext;
     TreeContextController = require('react-devtools-shared/src/devtools/views/Components/TreeContext')
       .TreeContextController;
+    InjectHookVariableNamesFunctionContext = require('react-devtools-shared/src/devtools/views/Components/InjectHookVariableNamesFunctionContext')
```

Comment:
This require statement is wrong. You're requiring the module but not extracting the context from it. Since you're using a default export in that file, I think you can fix by
```suggestion
    InjectHookVariableNamesFunctionContext = require('react-devtools-shared/src/devtools/views/Components/InjectHookVariableNamesFunctionContext').default;
```